src/common/hooks/useEndorsementChain/helpers.test.ts

```diff
@@ -0,0 +1,208 @@
+import { ethers } from "ethers";
+import { useProviderContext } from "../../contexts/provider";
+import { INFURA_API_KEY } from "../../../config";
+import { TradeTrustToken__factory } from "@govtechsg/token-registry/dist/contracts";
+import { fetchEventTime, getHolderOwner, mergeTransfers, sortLogChain } from "./helpers";
+import { TransferBaseEvent } from "../../../types";
+import { retrieveTitleEscrowAddressOnFactory } from "../useTitleEscrowContract";
+
+jest.mock("../../contexts/provider");
+
+const goerliProvider = new ethers.providers.InfuraProvider("goerli", INFURA_API_KEY);
+
+const mockUseProviderContext = useProviderContext as jest.Mock;
+
+describe("Test all endorsement chain helpers", () => {
+  beforeAll(() => {
+    mockUseProviderContext.mockReturnValue({ provider: goerliProvider, providerOrSigner: goerliProvider });
+  });
+
+  describe("test that fetchEventTime block time is as expected", () => {
+    it("test that fetchEventTime block 0 is as expected", async () => {
+      const timeOfFirstBlock = await fetchEventTime(0, goerliProvider);
+      expect(timeOfFirstBlock).toBe(1548854791000);
+    });
+
+    it("test that fetchEventTime block 1 is as expected", async () => {
+      const timeOfFirstBlock = await fetchEventTime(1, goerliProvider);
+      expect(timeOfFirstBlock).toBe(1548947453000);
+    });
+  });
```

Comment:
try to be concise with test statements, use `describe` to group, while test statements to start with `should... `
```suggestion
  describe("fetchEventTime", () => {
    it("should return correct formatted timestamp in milliseconds for block 0 with goerli provider", async () => {
      const timeOfFirstBlock = await fetchEventTime(0, goerliProvider);
      expect(timeOfFirstBlock).toBe(1548854791000);
    });

    it("should return correct formatted timestamp in milliseconds for block 1 with goerli provider", async () => {
      const timeOfFirstBlock = await fetchEventTime(1, goerliProvider);
      expect(timeOfFirstBlock).toBe(1548947453000);
    });
  });
```
just curious, why the need to convert to milliseconds?

Reply:
I think it's because the `new Date()` on `EndorsementChainLayout.tsx` uses milliseconds by default, so i just followed what was used previously

`format(new Date(data.timestamp ?? 0), "do MMM yyyy, hh:mm aa")`

`src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.tsx`